gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/StandardTableDefinition.java

```diff
@@ -252,12 +252,15 @@ ToStringHelper toStringHelper() {
   }
 
   @Override
-  public boolean equals(Object obj) {
-    return obj instanceof StandardTableDefinition && baseEquals((StandardTableDefinition) obj);
+  public final boolean equals(Object obj) {
```

Comment:
The `TableDefinition` subclasses probably also fall under the "final class, non-final equals/hashcode" case